src/WebJobs.Script/Workers/SharedMemoryDataTransfer/MemoryMappedFileAccessorUnix.cs

```diff
@@ -127,6 +136,64 @@ public override void Delete(string mapName, MemoryMappedFile mmf)
             }
         }
 
+        /// <summary>
+        /// Checks if a list of directories is specified in AppSettings to create <see cref="MemoryMappedFile"/>.
+        /// If one is specified, returns that list. Otherwise returns the default list.
+        /// </summary>
+        /// <returns>List of paths of directories where <see cref="MemoryMappedFile"/> are allowed to be created.</returns>
+        private List<string> GetAllowedDirectories()
+        {
+            string envVal = _environment.GetEnvironmentVariable(RpcWorkerConstants.FunctionsUnixSharedMemoryDirectories);
+            if (string.IsNullOrEmpty(envVal))
+            {
+                return SharedMemoryConstants.TempDirs;
+            }
+
+            return envVal.Split(',').ToList();
```

Comment:
Do we want to sanitize empty entries?
Do we want to log something like in the other?

Reply:
I see the logging is in the following, I guess that also covers the empty/non-valid entries.